System.IO.FileSystem/nanoFramework/StorageEventManager.cs

```diff
@@ -14,7 +14,7 @@ namespace nanoFramework.System.IO.FileSystem
     /// </summary>
     /// <param name="sender">Specifies the object that sent the Removable Device event. </param>
     /// <param name="e">Contains the Removable Device event arguments. </param>
-    public delegate void RemovableDeviceEventHandler(Object sender, RemovableDeviceEventArgs e);
+    public delegate void RemovableDeviceEventHandler(object sender, RemovableDeviceEventArgs e);
```

Comment:
Align filename with function(s) e.g. `RemovableDeviceEventManager.cs`

Reply:
This way, the old one can be deprecated and call this function instead.

Reply:
Good point... actually all this deserves a bit more thought. 
We probably want to make it clear that this is related with storage or media.

RemovableStorageEventManager?
RemovableMediaEventManager?
RemovableStorageDeviceEventManager?

Reply:
The ultimate question is are all drives removable, or is this more generic to storage devices like the original filename?!

Reply:
Not all are removable. There are internal storage "volumes" like serial flash/eeprom or even internal flash partitions. None of those use these "events".